airflow/models/dag.py

```diff
@@ -1522,3 +1522,46 @@ def create_dagrun(self,
                                             external_trigger=external_trigger,
                                             conf=conf,
                                             session=session)
+
+    @classmethod
+    def _find_dag_ids_including_subdags(cls, dag: DAG):
+        from airflow.operators.subdag_operator import SubDagOperator
```

Comment:
Can we move the import to top of the file, any reason of having it here?

Reply:
Same logic as https://github.com/apache/airflow/blob/master/airflow/models/dag.py#L1397 I assume?

Reply:
@kaxil this is to [avoid circular import](https://stackabuse.com/python-circular-imports/) and I added a comment.